src/brogue/RogueMain.c

```diff
@@ -1264,6 +1271,10 @@ void victory(boolean superVictory) {
         printHighScores(qualified);
     }
 
+    if (rogue.playbackMode && nonInteractivePlayback) {
+        printf("Recording: %s ended after %li turns (victory).\n", rogue.currentGamePath, rogue.playerTurnNumber);
```

Comment:
What if we also include escape vs mastery, and the amount of gold? Could be useful for collecting stats from recordings.

Reply:
See other comment